sqlup-mode.el

```diff
@@ -210,8 +224,10 @@ ANSI SQL keywords."
   (and (boundp 'sql-mode-font-lock-keywords) sql-mode-font-lock-keywords))
 
 (defun sqlup-keyword-p (word)
-  (cl-some (lambda (reg) (string-match-p (concat "^" reg "$") word))
-           (sqlup-keywords-regexps)))
+  (and (cl-some (lambda (reg) (string-match-p (concat "^" reg "$") word))
+                (sqlup-keywords-regexps))
+       (not (cl-some (lambda (blacklisted) (string-match-p (concat "^" (regexp-quote blacklisted) "$") word))
```

Comment:
I think this is separate from the question of whether this is an SQL keyword or not: it's a matter of whether or not we should upcase it. So I think the call to this (I dunno, `sqlup-excluded-keyword-p` maybe?) should probably be in `sqlup-capitablizable-p`.

Thoughts?